data/json/monster_special_attacks/monster_attacks.json

```
@@ -91,7 +91,7 @@
     "hit_dmg_u": "The %1$s stares at you intently and a sudden wind buffets you!",
     "hit_dmg_npc": "The %1$s stares intently at <npcname> and a hurricane force wind hits them!",
     "no_dmg_msg_u": "The %1$s's wind buffets your %2$s, but it doesn't catch hold!",
-    "no_dmg_msg_npc": "The %1$s's winds  hit <npcname>, but it doesn't catch hold!",
+    "no_dmg_msg_npc": "The %1$s's winds hits <npcname>, but it doesn't catch hold!",
```

```suggestion
    "no_dmg_msg_npc": "The %1$s's winds hit <npcname>, but it doesn't catch hold!",
```
I'm pretty certain this should stay as hit, but please confirm this either way.